### PR TITLE
Add accessToken to Redux Storage, Add constants

### PR DESCRIPTION
**Pull request by rawrrencee**
# Changelog:
Add constants for industry and categories/subcategories
Add accessToken to Redux Storage (previously only stored in localStorage)

## Checklist:

- [ ] Merged latest develop
- [ ] PR title makes sense

### CODE REVIEW

**Review by y1bs — DISMISSED** [review]
Seems ok! Thanks Lawrence!

**Review by y1bs — APPROVED** [review]

File: src/constants/information/index.js [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    categoryName: 'Music',
+    subcategories: [
+      'Instruments',
+      'Music Production',
+      'Music Fundamentals',
+      'Vocal',
+      'Music Techniques',
+      'Music Software',
+      'Other Music',
+    ],
```
Comment (natalieloke) [inline src/constants/information/index.js:271]: @rawrrencee can i check is the key supposed to be a string? `'health-fitness'` vs `music`

Comment (rawrrencee) [reply src/constants/information/index.js:271]: somehow the linting removes the apostrophe from the key automatically if its a single word :/

Comment (rawrrencee) [reply src/constants/information/index.js:271]: but we might need to restructure this array anyway, haven't used it yet so not sure what format we need it in

**Pull request merged**